### PR TITLE
Add kontena-cli 1.1.1

### LINKED ISSUE #1825: brew cask install kontena-cli
🍺  for the brave individual who makes this happen

### PR DESCRIPTION
**Pull request by kke**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference].
  * The cask is named **kontena-cli** as it only installs the CLI part of the stack, which is what most users want on their local machine as the rest of it will usually run on servers somewhere. In the future there may be installers for the server or agent components.
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not already refused in [closed issues].
- [X] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

(Fixes https://github.com/kontena/kontena/issues/1825)

### CODE REVIEW

**Comment by vitorgalvao** [comment]
This seems to be a CLI-only open-source tool. Please submit it first to [Homebrew/core](https://github.com/Homebrew/homebrew-core) and only if they don’t accept it, submit here (linking back to the pull request where it was refused).

This policy is documented in [faq/rejected_casks](https://github.com/caskroom/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> The app is both open-source and CLI-only (i.e. it only uses the `binary` artifact). In that case, and [in the spirit of deduplication](https://github.com/caskroom/homebrew-cask/issues/15603), submit it first to [Homebrew/core](https://github.com/Homebrew/homebrew-core). If it is rejected there, you may then try again in Homebrew-Cask (link us to the issue on Homebrew so we can see their reasoning for rejection).

File: Casks/kontena-cli.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'kontena-cli' do
+  version '1.1.1'
+  sha256 'a764f33f7b87ddbfcb844d09180956519c049b8b54c18967020741f17c39de96'
+
+  # github.com was verified as official when first introduced to the cask
```
Comment (neutric) [inline Casks/kontena-cli.rb]: The comment needs to [uniquely identify app or vendor](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment) (see the [`screencat`](https://github.com/caskroom/homebrew-cask/blob/5fc818752c30c156c00f79b04b66406189ab2f30/Casks/screencat.rb#L5) example). The URL should be `github.com/kontena/kontena`.

Comment (kke) [inline Casks/kontena-cli.rb]: That's fixed in the fork but it's not updating here as the PR is closed.

Comment (neutric) [inline Casks/kontena-cli.rb]: Sorry for the comment after closing, overlapped with Vitor's assessment, which makes mine null and void anyway.

**Pull request closed without merging**